### PR TITLE
Build files for Database library

### PR DESCRIPTION
**Pull request by FuriousLlama**
Added Azure build files. Added stylecop exclusion to the migration files

### CODE REVIEW

**Comment by healthgatewaybc** [comment]
SonarQube Report: https://sonarqube-gateway.pathfinder.gov.bc.ca/dashboard?id=Medicationservice_pr_168

File: Apps/Database/src/Migrations/20191024233502_InitialCreate.cs [hunk 1]
```diff
@@ -457,3 +460,7 @@ protected override void Down(MigrationBuilder migrationBuilder)
         }
     }
 }
+
+
+#pragma warning restore SA1200 // Using directive should appear within a namespace declaration
```
Comment (bradhead) [inline Apps/Database/src/Migrations/20191024233502_InitialCreate.cs]: will we ever need to regen these initial create files and so if so will we lose these code-smell-avoiding pragmas?

Comment (FuriousLlama) [reply Apps/Database/src/Migrations/20191024233502_InitialCreate.cs]: Correct. Fortunately it looks like we only need to add // <auto-generated/> to the first line of the files

**Comment by healthgatewaybc** [comment]
SonarQube Report: https://sonarqube-gateway.pathfinder.gov.bc.ca/dashboard?id=Medicationservice_pr_168

**Pull request merged**